docs-preview/media/styles.css

```diff
@@ -0,0 +1,104 @@
+body, html {
+  margin-left:auto;
+  margin-right: auto;
+  max-width: 967px;
+  padding: 0;
+  font-size: 16px !important;
```

Comment:
When comparing the sizes of font in the developer console, they seem to be the same, but visually on the screen, I seems like the font should be smaller. 16px font-size does not match the published output compared to the markdown preview visually.